packages/back-end/src/routers/project/project.controller.ts

```diff
@@ -281,7 +280,7 @@ export const deleteProject = async (
 
   // Clean up saved groups
   try {
-    await removeProjectFromSavedGroups(id, org.id);
+    await context.models.savedGroups.removeProject(id);
```

Comment:
Nit: the name of this method makes the id argument a little ambiguous. Looking at this code, I might think you are supposed to pass in the saved group Id and not the project id. Maybe something like `removeProjectIdFromAllGroups`would be clearer?